[ar1_processes] to [eigen_II] [heavy_tails][inequality]Example and spelling

longye-tian opened the pull request:
Dear John @jstac and Matt @mmcky ,

This pull request is to check the example admonition and spelling of from lecture [ar1_processes.md] to [eigen_II.md] according to #520 and #532.  

The details are shown below:

- ar1_processes.md
   - add two example admonitions
- business_cycle.md (no typos, no example admonition)
- cagan_adaptive.md (no typos, no example admonition)
- cagan_ree.md
   - change 'a central banks's' to 'a central bank's'
   - change 'fisal' to 'fiscal'
   - change " to start a persistent inflation the government 'beings' " to  'begins'
- cobweb.md (no typos, no example admonition)
- commod_price.md (no typos, no example admonition)
- complex_and_trig.md
   - add one example admonition
- cons_smooth.md
   - change 'consumption smoothing model' to 'consumption-smoothing model' for consistency
   - change 'first order' to 'first-order' for consistency
   - change 'finanical' to 'financial'
   - change 'accross' to 'across'
   - change 'enlighening' to 'enlightening'
   - change the Capitalized title to match the style guide
   - change 'create a function that generate' to 'create a function that generates'
 - eigen_I.md
   - add one example admonition
 - eigen_II.md
   - change the subtitle to lower case to match the style guide
   - change 'Perron-Frobenius Theorem' to 'Perron-Frobenius theorem' for consistency (used in the current subtitles and also in wikipedia)
   - add example admonition
 - heavy_tails.md
   - change 'from world population review' to 'from the World Population Review'
   - change 'Law of Large Numbers' to 'law of large numbers' to match with Wikipedia form as discussed before
   - change 'heavy tailed' to 'heavy-tailed' for consistency
   - change 'The value the tail index' to 'The value of the tail index'
   - change 'Here is code that generates' to 'Here is the code generates'
   - add two example admonitions
- inequality.md
  - change 'per capital' to 'per capita'
  - change 'others is zero' to 'others are zero'
  - change 'the world bank' to 'the World Bank' to match Wikipedia 
  - add empty link to match with the style guide
  - change 'Western economies GDP' to 'Western economies'GDP'
  - add one example admonition
- input_output.md
  - change 'input output' to 'input-output' for consistency
  - change **feasible allocations must satisfy**  to *feasible allocations must satisfy* to match with style guide
  - change 'a two good economy' to 'a two-good economy'  
  - add two example admonitions containing code blocks.

Best ❤️ 
Longye

longye-tian committed:
update_ar1_example

update_ar1_example;
no spelling error.

longye-tian committed:
update cagan_ree typos

This commit updates cagan_ree.md typos

longye-tian committed:
Update complex_and_trig.md

add example admonition

longye-tian committed:
Merge branch 'main' into example_spelling

longye-tian committed:
Update cons_smooth.md

update typos

longye-tian committed:
Merge branch 'example_spelling' of https://github.com/QuantEcon/lecture-python-intro into example_spelling

longye-tian committed:
Update eigen_II.md

update typos

longye-tian committed:
Update eigen_II.md

add example admonition

longye-tian committed:
Update eigen_I.md

add example admonition

jstac commented:
Many thanks @longye-tian .  This is very helpful.

@mmcky , please review and merge when ready.

longye-tian committed:
update typos in heavy_tails.md

   - change 'from world population review' to 'from the World Population Review'
   - change 'Law of Large Numbers' to 'law of large numbers' to match with Wikipedia form as discussed before
   - change 'heavy tailed' to 'heavy-tailed' for consistency
   - change 'The value the tail index' to 'The value of the tail index'
   - change 'Here is code that generates' to 'Here is the code generates'

longye-tian committed:
[heavy_tails]add two example admonitions

longye-tian committed:
[inequality] typo and grammar update

  - change 'per capital' to 'per capita'
  - change 'others is zero' to 'others are zero'
  - change 'the world bank' to 'the World Bank' to match Wikipedia
  - add empty link to match with the style guide
  - change 'Western economies GDP' to 'Western economies'GDP'

longye-tian committed:
[inequality] add one example admonition

longye-tian committed:
[input_output] update typos

  - change 'input output' to 'input-output' for consistency
  - change **feasible allocations must satisfy**  to *feasible allocations must satisfy* to match with style guide
  - change 'a two good economy' to 'a two-good economy'

longye-tian committed:
[input_output] example admonition

This example admonition include code  block

longye-tian committed:
[input_output] example admonition update

cannot include code blocks in the example admonition

longye-tian committed:
Merge branch 'main' into example_spelling

longye-tian committed:
Merge branch 'main' into example_spelling

mmcky reviewed (approved):
thanks @longye-tian this is great work.
